### PR TITLE
moved CGI credentials to nf secrets

### PR DESCRIPTION
**Pull request by HomoPolyethylen**
CGI credentials used to be pipeline param, moved to nextflow secrets, so that credentials are not open readable.

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/variantmtb/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/variantmtb _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by HomoPolyethylen** [comment]
`nf-core CI / Run pipeline with test data (21.10.3)` fails:

<details><summary>Error mesage</summary>
<p>

```terminal
Error executing process > 'NFCORE_VARIANTMTB:VARIANTMTB:QUERYNATOR_CIVICAPI (test_sample)'

Caused by:
  Failed to create Conda environment
  command: conda create --mkdir --yes --quiet --prefix /home/runner/work/variantmtb/variantmtb/work/conda/env-8333bfb82ced3d481ee8f58aa4d2aef8 bioconda::querynator=0.4.1
  status : 1
  message:
    PackagesNotFoundError: The following packages are not available from current channels:
    
      - bioconda::querynator==0.4.1
    
    Current channels:
    
      - defaults
      - https://conda.anaconda.org/bioconda
```
</p>
</details> 

had this issue before, if I remember correctly, adding `conda-forge` resolved it

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,16 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### `Added`
 
-- [#1](https://github.com/qbic-pipelines/variantmtb/pull/1) - Query to CGI & CIViC. Creation of a comprehensive HTML report. 
-
+- [#1](https://github.com/qbic-pipelines/variantmtb/pull/1) - Query to CGI & CIViC. Creation of a comprehensive HTML report.
```
Comment (mapo9) [inline CHANGELOG.md:10]: did you change anything here?
I think we dont have to but only create a changelog when we push to master. However, maybe we should save everything we change somewhere

File: modules/local/querynator/cgiapi.nf [hunk 2]
```diff
@@ -43,11 +45,11 @@ process QUERYNATOR_CGIAPI {
...
         --cancer $cancer \\
         --genome $genome \\
-        --token $token \\
-        --email $email \\
+        --token \${cgi_token} \\
+        --email \${cgi_email} \\
         --filter_vep \\
         $args
-    
+
```
Comment (mapo9) [inline modules/local/querynator/cgiapi.nf:53]: very nice! 💪

File: CHANGELOG.md (hunk 1 above)
Comment (HomoPolyethylen) [reply CHANGELOG.md:10]: seems like my prettier run did this

Comment (mapo9) [reply CHANGELOG.md:10]: ah ok. sure looks like it

File: modules/nf-core/bcftools/norm/environment.yml
Comment (famosab) [inline modules/nf-core/bcftools/norm/environment.yml:1]: 👍

**Review by famosab — APPROVED** [review]
Looks good to me.

We just need to fix the CI stuff. Do we know why Python 3.6 is used?

**Review by mapo9 — APPROVED** [review]
thanks Famke. 
Niclas, maybe you can add the problems that occured to the issue that already exists with the ToDos
LGTM then 🚀

**Review by mapo9 — APPROVED** [review]
nice fix!
LGTM 
let's merge

**Pull request merged**